### DISCUSSION TITLE
default mappings

### DISCUSSION THREAD

**Discussion by aloispichler** [ggandor/leap.nvim#193] [General]
The default mappings are
`s` for `<Plug>(leap-forward-to)` and
`S` for `<Plug>(leap-backward-to)`.
To me, this is not natural and counterintuitive, because the `g` motions (`gg` and `G`) are opposite: lowercase goes up, uppercase down. So, I suggest to change this behavior (but this is only my opinion).

**Comment by ggandor** [comment]
`s`/`S` is consistent with `/`/`?`, `n`/`N`, and `f`/`F`.
